R/as_html.R

```diff
@@ -210,10 +214,13 @@ as_html <- function(x,
   ## we want them to be there but empty??
   ftrlst <- list(
     if (length(mat$ref_footnotes) > 0) rfnotes,
+    if (length(mat$ref_footnotes) > 0) hsep_line,
     if (length(main_footer(x)) > 0) mftr,
+    if (length(main_footer(x)) > 0 && length(prov_footer(x)) > 0) tags$br(),
```

Comment:
this is blank row right? we might want to be more explicit with names (I am trying to do the same myself), so things are more readable

Reply:
I left a comment to specify that it's a line break